xenoglossia/builtins.py

```diff
@@ -153,6 +153,17 @@ def title(input, *args):
     """
     return input.title()
 
+@xenoglossia_fn
+@string_fn
+def flip(input, *args):
+    """
+    Returns an upside down version of *input* prepended by our favorite Flip Table dude.
+    """
+    from_chars = " abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
+    to_chars = u" \u0250q\u0254p\u01DD\u025F\u0183\u0265\u1D09\u027E\u029El\u026Fuodb\u0279s\u0287n\u028C\u028Dx\u028Ez\u2200q\u0186p\u018E\u2132\u05E4HI\u017F\u029E\u02E5WNO\u0500Q\u0279S\u2534\u2229\u039BMX\u2144Z"
+    char_dict = dict(zip(from_chars, to_chars))
+    flipped_input = "".join(map(lambda (c): char_dict[c], input)[::-1])
```

Comment:
Minor nit: instead of `char_dict[c]`, would be good to do `char_dict.get(c, c)` - that returns the original character as the default if the character being fetched isn't in `char_dict`.